packages/node_modules/@ciscospark/plugin-people/src/config.js

```diff
@@ -6,6 +6,9 @@ export default {
   people: {
     batcherWait: 100,
     batcherMaxCalls: 10,
-    batcherMaxWait: 1500
+    batcherMaxWait: 1500,
+    // optional flag that requires Hydra to send every type field,
```

Comment:
Use jsdoc formatted comments. config isn't well documented yet, but check a few of the other plugin configs and you should find one that has all the required doc tags.

Reply:
Done.